### DISCUSSION TITLE
How to get $events/client_connected message by MQTT Bridge

### DISCUSSION THREAD

**Discussion by Tuxzx** [emqx/emqx#8491] [Q&A]
I'm following the [data-bridge-document](https://www.emqx.io/docs/en/v5.0/data-integration/rule-sql-grammar-and-examples.html#sql-statement-example) to try to get some client events like $events/client_connected.

From the dashboard, I can see the event is sent, but the logs file shows out of service.  I have not gotten any messages.

Should I do some config for this?

### action
<img width="1068" alt="截屏2022-07-15 下午4 39 44" src="https://user-images.githubusercontent.com/29911719/179187368-ada36252-fc47-451b-81bf-0db14d1c29c2.png">

### data bridge
<img width="882" alt="截屏2022-07-15 下午4 39 22" src="https://user-images.githubusercontent.com/29911719/179187383-011efe33-171d-475d-a6b4-5c924f3d16f1.png">

### status
<img width="1101" alt="截屏2022-07-15 下午4 48 41" src="https://user-images.githubusercontent.com/29911719/179188785-1f8b2694-f364-4333-9558-dc2e7962439a.png">

## logs
```
2022-07-15T16:16:57.781548+08:00 [warning] action: <<"mqtt:mysession">>, clientid: hello2, line: 333, mfa: emqx_rule_runtime:handle_action/4, msg: out_of_serv
ice, peername: 127.0.0.1:59163
2022-07-15T16:17:00.454590+08:00 [warning] action: <<"mqtt:mysession">>, clientid: hello2, line: 333, mfa: emqx_rule_runtime:handle_action/4, msg: out_of_serv
ice, peername: 127.0.0.1:59174
```

## log detail
<img width="772" alt="截屏2022-07-15 下午5 20 30(1)" src="https://user-images.githubusercontent.com/29911719/179194540-86f0e007-fdb9-41a4-b3f6-0faf6731220a.png">
<img width="532" alt="截屏2022-07-15 下午5 20 49" src="https://user-images.githubusercontent.com/29911719/179194554-12b95c20-bffe-417f-b14e-4560b487ae49.png">

**Comment by Tuxzx** [comment]
It seems not match.

<img width="560" alt="截屏2022-07-15 下午5 25 52" src="https://user-images.githubusercontent.com/29911719/179196057-13be11fb-45c5-4922-bdf8-bd13ddf643d5.png">

**Reply by lafirest** [reply]
https://github.com/emqx/emqx/issues/8494
Seems to be the same problem
